src/TextField/TextField.jsx

```diff
@@ -84,6 +83,11 @@ export const TextField = ({
 }
 
 TextField.propTypes = {
+  /**
+   * Overrides the styles applied to the component.
+   */
+  autoComplete: PropTypes.string,
```

Comment:
My preference would be to turn this into a boolean, rather than a string. The text field component can take care of translating this into a `on` or `off` value.

Reply:
Ah! 

It's intentionally a string: you can give helper information to the browsers by setting different values:

https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete-nickname

Reply:
> Yep, totally agree.

I've rebased out the `Autocomplete` component commit.

Reply:
TIL

Reply:
For 99% of the cases, a boolean would've been fine. I'm thinking there might be specific situations (like adding a country selector) where we might have to be more specific with it.

This bit made me 🤯:

The "on" keyword indicates that the user agent is allowed to provide the user with autocompletion values, but does not provide any further information about what kind of data the user might be expected to enter. **User agents would have to use heuristics to decide what autocompletion values to suggest.**